### ISSUE TITLE
DHP19EPC_dataset

### ISSUE THREAD

**Issue by efjksfef** [MasterHow/EventPointPose#6]
What should I do with the downloaded dataset?
root_train_data_dir = 'W:\\模型\EventPointPose-main\\DHP19EPC_dataset\\train_MeanLabel_extract'
   

     # path of valid data
        root_valid_data_dir = 'W:\\模型\EventPointPose-main\\DHP19EPC_dataset\\test_MeanLabel_extract'

        train_dataset = DHP19EPC(
            args,
            root_data_dir=root_train_data_dir + 'data//',
            root_label_dir=root_train_data_dir + 'label//',
            root_3Dlabel_dir=root_train_data_dir + '3Dlabel//',
            root_dict_dir=root_train_data_dir + 'Point_Num_Dict.npy',

What should be filled in for these paths above?
